app/assets/stylesheets/taxonomies.scss

```diff
@@ -0,0 +1,24 @@
+$style-none: none; 
+
+#taxonomies {
+  .taxons-list {
+    list-style-type: $style-none;
+    top: 20px;
+    padding-left: 0;
+
+    li a:hover {
```

Comment:
When adding styling to an anchor could you please use a class instead of styling the element directly 👍